Gemfile

```diff
@@ -8,14 +8,15 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
-
+gem 'devise'
 gem 'coffee-rails', '~> 4.2'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'webpacker', '~> 3.5'
 gem 'tailwindcss', '~> 0.2.0'
-gem "haml-rails", "~> 1.0"
+gem 'omniauth', '~> 1.6', '>= 1.6.1'
```

Comment:
why do you add this gem?

Reply:
We are planning on adding login with LinkedIn later that's why we added it, but since we do not need it right now we will remove it.